services/collections-publisher/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.7'
+
+x-collections-publisher: &collections-publisher
+  build:
+    context: .
+    dockerfile: services/collections-publisher/Dockerfile
+  image: collections-publisher
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+    - home:/home/build
+  working_dir: /govuk/collections-publisher
+
+services:
+  collections-publisher-lite:
+    <<: *collections-publisher
+    privileged: true
+    depends_on:
+      - mysql
+      - redis
+    environment:
+      # the app uses this URL to generate a test database URL
+      DATABASE_URL: "mysql2://root:root@mysql/collections_publisher_development"
+      REDIS_URL: redis://redis
+      DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: "true"
+
+  collections-publisher-app: &collections-publisher-app
+    <<: *collections-publisher
+    depends_on:
+      - mysql
+      - redis
+      - content-store-app
+      - link-checker-api-app
+      - nginx-proxy
+      - publishing-api-app
```

Comment:
did we want the worker to appear here too (so adding `- collections-publisher-worker`)?

Reply:
yes, thanks - it was there before... I think I must have done a wonky rebase and it got knocked off!